pacman/executor/algorithm_decorators/token.py

```diff
@@ -31,6 +31,7 @@ def __hash__(self):
         return str(self._name + ":" + str(self._part)).__hash__()
 
     def __eq__(self, other):
+        # pylint: disable=protected-access
```

Comment:
obviously the correct fix here is to have porperties for those privates, not just letting protected stuff be circumvented

Reply:
This is accessing items that are part of the class, which has been type-checked, so it is fine here

Reply:
Agreed with @rowleya; if this was Java or C#, we wouldn't have the complaints at all.